src/main/java/io/th0rgal/oraxen/mechanics/provided/gameplay/stringblock/StringBlockMechanicListener.java

```diff
@@ -170,42 +178,32 @@ public void onBreakingCustomBlock(final BlockBreakEvent event) {
         final Block blockAbove = block.getRelative(BlockFace.UP);
         final Player player = event.getPlayer();
         for (BlockFace face : BlockFace.values()) {
-            if (face == BlockFace.SELF && !face.isCartesian()) continue;
+            if (face == BlockFace.SELF) continue;
```

Comment:
isCartesian was used to make sure the blockface was only NSWE not NW or SW, this should probably be left in